client/my-sites/purchases/subscriptions/subscriptions-content.tsx

```diff
@@ -36,7 +36,7 @@ function SubscriptionsContent( {
 	selectedSite: undefined | null | SiteDetails;
 	purchases: Purchase[];
 } ) {
-	const getManagePurchaseUrlFor = ( siteSlug: string, purchaseId: number ) =>
+	const getManagePurchaseUrlFor = ( siteSlug: string, purchaseId: string | number ) =>
```

Comment:
In this case, I am not sure if we need to include a number type. This version isn't calling `managePurchase`, since it needs the URL root to be `purchases/subscriptions/` instead of `me/purchases`.

Thoughts on what to do here?